docs/en/latest/USER_GUIDE.md

```diff
@@ -23,31 +23,31 @@ title: User Guide
 
 The following are parts of the modules' snapshot.
 
-## Metrics
+## Dashboard
 
 We support the monitor page by referencing it in [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe). Before accessing [Grafana](https://grafana.com/), please Enable [`allow_embedding=true`](https://grafana.com/docs/grafana/latest/administration/configuration/#allow_embedding), which defaults to `false`. This causes the browser to fail to render Grafana pages properly due to security policies.
 
-![metrics-en](../../assets/images/metrics-en.png)
+![Dashboard-en](https://user-images.githubusercontent.com/40708551/112922395-0eed0380-912a-11eb-8c92-4c67d2bae4a8.png)
```

Comment:
@iamayushdas thank you for this. I was wondering that where the new image file is located.

Reply:
GitHub CDN?

Reply:
> GitHub CDN?

Yep, absolutely I think. My concern is how could us maintain these images. Previous images were maintained in this repository, while the new images seem not.

Reply:
> @iamayushdas thank you for this. I was wondering that where the new image file is located. 

My pleasure sir!

Reply:
🤔 that's a problem.

- If we put them in our repo, it's easy to maintain actually, but will make the `pull` action slower;
- if we put them in the CDN, we need to upload them when updating the images, but will speedup the pull action.

Both are ok to me, how about others' opinion?

Reply:
I think CDN gives us more ease to handle

Reply:
😄

Reply:
> I think CDN gives us more ease to handle

@iamayushdas agreed +1 on Github CDN. Actually I'm not sure that is there any possibility for the images be unavailable to the pages. For example, the Github account which uploaded the images is deleted, or any other reasons.

It's ok for me if the URLs of images will be never changed or inaccessible. Thanks.

Reply:
No, there wont be a problem because its under the github server and deleting the account won't create any isse. 
Now if your are fine with this PR you can give a approval to it :smiley:

Reply:
> No, there wont be a problem because its under the github server and deleting the account won't create any isse. 
> 
> Now if your are fine with this PR you can give a approval to it :smiley:  @imjoey 

@iamayushdas Sure, very glad to know it will always work. Thanks for clarification.

Reply:
☺️ #OpenSourceIsLove

Reply:
Hii all, want to add something. We could also go for `git lfs` (faster pull). The CDN approach is okay until we need to update those static files in future releases where pinpointing the content through its CDN URL will be a huge pain in the neck. On the other hand, relative links .... (you know, just update the static file, that's it, hehe) :)

Reply:
but don't you think we should only go for that if we are handling large files?

Reply:
Haha, there is no exact threshold to consider what is a large file, isn't it!! Just said about another available option with better docs management (IMO) :)

Reply:
I agree 😊